### PR TITLE
Facet selection homepage improvements

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
This PR addresses the request raised on this other PR: https://github.com/bitcoinsearch/bitcoinsearch-app/pull/60#issuecomment-1448896295

It should be merged after https://github.com/bitcoinsearch/bitcoinsearch-app/pull/60 is merged.

### CODE REVIEW

**Review by adamjonas — COMMENTED** [review]
When you click on an author the keywords and authors stick around, but when you click on a keyword they disappear. They should both function like the keyword.

https://user-images.githubusercontent.com/755825/223244586-50a1fadc-fe06-49a7-b1a7-a463769abd3b.mov

File: src/components/filterTag/FilterTags.jsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
+import { Button } from "@chakra-ui/react";
 import { withSearch } from "@elastic/react-search-ui";
 import React from "react";
 import "./filterTags.scss";
 
 const FilterTags = ({ filters, addFilter, removeFilter, field, options }) => {
   const onRemove = (value) => {
-    removeFilter(field, value, "all");
+    removeFilter(field, value, "any");
```
Comment (Emmanuel-Develops) [inline src/components/filterTag/FilterTags.jsx:8]: this is where I changed the filter behaviour. You can revert it so the sidebar and filter pills both have the same filter behaviour

**Comment by Emmanuel-Develops** [comment]
@BalogunofAfrica The author pills are disconnected from the author's side bar facet. I did this for the initial behavior where users can select a combination of pills (union rather than an intersection) using `addFilter` and `removeFilter` type of all.

But given that the behaviour has changed selecting any pills on the hardcoded values hide all the pills then kindly revert to the previous

https://user-images.githubusercontent.com/62019510/223821935-67d871fc-b9ac-4693-8753-071ca79fbacc.mov

File: src/components/filterTag/FilterTags.jsx (hunk 1 above)
Comment (BalogunofAfrica) [reply src/components/filterTag/FilterTags.jsx:8]: Oh seen, would resolve now

File: src/components/homeFacetSelection/index.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  results,
+  isLoading,
+}) => {
+  const isSearched = Boolean(
+    results.length &&
+      (resultSearchTerm?.trim() || (filters.length && results?.length))
+  );
+
+  const onRemove = (value) => {
+    removeFilter(field, value, "any");
```
[the remark this answers is not in the document]
Comment (BalogunofAfrica) [reply src/components/homeFacetSelection/index.jsx]: @Emmanuel-Develops  Done

**Comment by adamjonas** [comment]
@BalogunofAfrica 🎉

**Pull request merged**